### PR TITLE
NP-2474: create defaultConstructor

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: customer/src/main/java/no/unit/nva/customer/get/GetControlledVocabularyHandler.java [hunk 1]
```diff
@@ -40,8 +61,19 @@ protected Integer getSuccessStatusCode(Void input, VocabularySettingsList output
...
-    @Override
-    protected List<MediaType> listSupportedMediaTypes() {
-        return SUPPORTED_MEDIA_TYPES;
+    private static CustomerService defaultCustomerService() {
+        AmazonDynamoDB client = defaultDynamoDbClient();
+
+        return new DynamoDBCustomerService(client, ObjectMapperConfig.objectMapper, ENVIRONMENT);
+    }
+
+    private static AmazonDynamoDB defaultDynamoDbClient() {
```
Comment (axthosarouris) [inline customer/src/main/java/no/unit/nva/customer/get/GetControlledVocabularyHandler.java:70]: I think default amazon clients should be in commons. I want to test how much of a change this configuration actually does.

File: customer-commons/src/main/java/no/unit/nva/customer/Constants.java [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 
 public final class Constants {
 
+    public static final Environment ENVIRONMENT = new Environment();
     public static final List<MediaType> DEFAULT_RESPONSE_MEDIA_TYPES = List.of(
         MediaType.JSON_UTF_8,
         MediaTypes.APPLICATION_JSON_LD
     );
 
+    public static final String DEFAULT_AWS_REGION = "eu-west-1";
```
Comment (axthosarouris) [inline customer-commons/src/main/java/no/unit/nva/customer/Constants.java:17]: It is advised to set these things up a-priori. I want to test if it works

File: customer-commons/src/main/java/no/unit/nva/customer/Constants.java [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
+    public static final String AWS_REGION = ENVIRONMENT.readEnvOpt("AWS_REGION").orElse(DEFAULT_AWS_REGION);
```
Comment (axthosarouris) [inline customer-commons/src/main/java/no/unit/nva/customer/Constants.java:18]: AWS_REGION is always included as an evn variable by default.

File: customer/src/main/java/no/unit/nva/customer/get/GetControlledVocabularyHandler.java [hunk 4]
```diff
@@ -1,31 +1,52 @@
...
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
 
 public class GetControlledVocabularyHandler extends ApiGatewayHandler<Void, VocabularySettingsList> {
 
     public static final String IDENTIFIER_PATH_PARAMETER = "identifier";
+    public static final Environment ENVIRONMENT = new Environment();
     private static final List<MediaType> SUPPORTED_MEDIA_TYPES =
-        List.of(MediaType.JSON_UTF_8,MediaTypes.APPLICATION_JSON_LD);
+        List.of(MediaType.JSON_UTF_8, MediaTypes.APPLICATION_JSON_LD);
```
Comment (brinxmat) [inline customer/src/main/java/no/unit/nva/customer/get/GetControlledVocabularyHandler.java:31]: It's exciting to have a lot of warnings

**Review by brinxmat — APPROVED** [review]

**Pull request merged**